tests/integration/steps.go

```diff
@@ -20,6 +20,8 @@ var happyPathSteps = concatSteps(
 	stepsUnbond("consu"),
 	stepsRedelegate("consu"),
 	stepsDowntime("consu"),
+	stepsRejectEquivocationProposal("consu", 2),
+	stepsDoubleSign("consu"),
```

Comment:
Why would a double sign on the provider allow the equivocation prop to pass? Isn't it a consumer double sign that we're looking for? 

Maybe I'm misunderstanding something

Reply:
`stepsDoubleSign` first double signs on the provider and then on the consumer.